en/tools/config-catalog-generator/data/configs.json

```diff
@@ -1761,6 +1761,1416 @@
+                            "default": "60000",
+                            "possible": "3000",
+                            "description": ""
+                        },
+                        {
+                            "name": "acceptCount",
+                            "type": "integer",
+                            "required": true,
+                            "default": "200",
+                            "possible": "-",
+                            "description": ""
+                        },
+                        {
+                            "name": "server",
+                            "type": "string",
+                            "required": true,
+                            "default": "WSO2 Carbon Server",
+                            "possible": "-",
+                            "description": ""
+                        },
+                        {
+                            "name": "compression",
+                            "type": "string",
+                            "required": true,
+                            "default": "on",
+                            "possible": "-",
+                            "description": ""
+                        },
+                        {
+                            "name": "compressionMinSize",
+                            "type": "integer",
+                            "required": true,
+                            "default": "2048",
+                            "possible": "-",
+                            "description": ""
+                        },
+                        {
+                            "name": "noCompressionUserAgents",
+                            "type": "string",
+                            "required": true,
+                            "default": "gozilla, traviata",
+                            "possible": "-",
+                            "description": ""
+                        },
+                        {
+                            "name": "compressibleMimeType",
+                            "type": "string",
+                            "required": true,
+                            "default": "text/html,text/javascript,application/x-javascript,application/javascript,application/xml,text/css,application/xslt+xml,text/xsl,image/gif,image/jpg,image/jpeg",
+                            "possible": "-",
+                            "description": ""
+                        },
+                        {
+                            "name": "URIEncoding",
+                            "type": "string",
+                            "required": true,
+                            "default": "UTF-8",
+                            "possible": "-",
+                            "description": ""
+                        }
+                    ]
+                }
+            ],
+            "exampleFile": "_http-transport.toml"
+        },
+        {
+            "title": "HTTPS Transport (catalina-server)",
```

Comment:
Similarly may be `HTTPS Servlet Transport`?

Reply:
will change

Reply:
resolved